.github/workflows/codeql.yml

```diff
@@ -0,0 +1,108 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "main", "release-*" ]
+  pull_request:
+    branches: [ "main", "release-*" ]
+  schedule:
+    - cron: '27 4 * * 1'
```

Comment:
What's the significance of 4:27?

Reply:
This was the default when GH created the file for me.  Maybe random, or the time I created the file?  Can change to a more "standard" time if we want.